mkdocs.yml

```diff
@@ -6,6 +6,9 @@ plugins:
         on_pre_build: "docs.hooks:pre_build"
 repo_url: 'https://github.com/aeternity/aepp-sdk-js'
 
+theme:
+  name: readthedocs
```

Comment:
Please, don't mix the changes from different PRs

Reply:
ah, that one slipped my attention.